prow/config/external_plugins_config.yaml

```diff
@@ -931,6 +931,24 @@ ti-community-format-checker:
           <sub>:open_book: For more info, you can check the ["Contribute Code"](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/contribute-code.html#referring-to-an-issue) section in the development guide.</sub>
         missing_label: do-not-merge/needs-linked-issue
         skip_label: skip-issue-check
+      - pull_request: true
+        body: true
+        branches: [master]
+        regexp: "\\\\n\\\\bTests <!-- At least one of them must be included\\\\. -->\\\\s*[\\\\n]{1,}(- \\\\[[xX ]\\\\] .*\\\\n)*(- \\\\[[xX]\\\\] .*\\\\n)(- \\\\[[xX ]\\\\] .*\\\\n)*\\\\n"
+        missing_label: do-not-merge/needs-finished-tests
+        missing_message: |
+          **Notice**: To remove the `do-not-merge/needs-finished-tests` label, please finished the test for the pull request and check the finished items in description.
```

Comment:
> `do-not-merge/needs-finished-tests`

How about renaming it to `do-not-merge/needs-tests-checked`? IMO it's more straightforward.